thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart/component.js

```diff
@@ -30,19 +31,30 @@ export default Ember.Component.extend({
     show: false
   },
 
+  buildTooltip: buildTooltip,
```

Comment:
JSDoc? It'd be great to add as much as we can.

Reply:
:+1: This is basically importing the template helper defined earlier. Will add a note with my following pr.